aten/src/ATen/native/cuda/ActivationSiluKernel.cu

```diff
@@ -47,7 +47,7 @@ void silu_backward_kernel(TensorIteratorBase& iter) {
           const opmath_t dy_acc = static_cast<opmath_t>(dy);
           const opmath_t x_acc = static_cast<opmath_t>(x);
           const opmath_t s_acc =
-              opmath_t(1) / (opmath_t(1) + c10::cuda::compat::exp(-x_acc));
+              opmath_t(1) / (opmath_t(1) + c10::complex<double>::cexp(-x_acc));
```

Comment:
```suggestion
              opmath_t(1) / (opmath_t(1) + c10::cuda::compat::exp(-x_acc));
```

Reply:
Oh, sorry. I misunderstood. I changed the code

https://github.com/pytorch/pytorch/pull/89649/commits/44720c41641741d8298d2a15e5f0c80fe06b7235